LDRH_choice.py

```diff
@@ -460,8 +466,8 @@ def run_staircase(task, operation=None):
 
 
 ## QUESTIONNAIRE ##
-Questions = task_function.questionnaire(win)
-if Questions.run_questionnaire(win)=='QUIT': pickle_and_quit()
+# Questions = task_function.questionnaire(win)
```

Comment:
Hi Richard,

The images arent missing. They are just excluded on gitignore. That's why its not sync into the github. That's why it shouldn't be opened as an issue either. we never made missing audio an issue because its not on github? we usually sync the image and sound files directly to github.

Also, the questionnaire code are temporarily initialized on that line for testing purposes. (It will be initialized at the end of the task otherwise). Please do not delete the codes on that line. 

Regards,
Ratnaganadi Paramita
Center for Human Development
University of California San Diego
AP&M 5444 | rparamit@ucsd.edu

On Jul 9, 2015, at 7:22 PM, Richard Tibbles <notifications@github.com> wrote:

In LDRH_choice.py:

> @@ -460,8 +466,8 @@ def run_staircase(task, operation=None):
>  
>  
>  ## QUESTIONNAIRE ##
> -Questions = task_function.questionnaire(win)
> -if Questions.run_questionnaire(win)=='QUIT': pickle_and_quit()
> +# Questions = task_function.questionnaire(win)
Let's open this as an issue.

—
Reply to this email directly or view it on GitHub.